modules/javafx.web/src/main/java/javafx/scene/web/WebView.java

```diff
@@ -1282,6 +1287,8 @@ private void registerEventHandlers() {
                }
            });
         setOnDragDone(event -> {
+                mimes = null;
+                values = null;
```

Comment:
Is it possible for `OnDragDone` to be missed? If so, I wonder if it is a good idea to also set `mimes` and `values` to `null` in the `OnDragDetected` handler?

Reply:
It makes sense. Done.